protocol/protobuf/contact_verification.pb.go

```diff
@@ -1,245 +1,380 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.27.1
+// 	protoc        v3.12.4
```

Comment:
please, set up a correct version:
1. removed all my manual go-related installations (like protobuf and etc)
2. run make setup , make setup-dev
3. It will generate ~/go folder. Add path  ~/go/bin to PATH
4. Done, version should be the correct

Reply:
Okay thanks, let me perform this steps and resend a revision after testing. I understand the expectations should be that we don't have the protobuf files. Hopefully this decreases the number of files.

Reply:
you will have some one, but not so much and no with these changes. At least you should not have this lines with the version